index.html

```diff
@@ -12,7 +12,7 @@
 
 <body>
   <nav class="navbar">
-    <h2>My Logo</h2>
+    <h2 class="mylogo">My Logo</h2>
     <img class="icon" src="images/Union.jpg" alt="Union">
   </nav>
   <section class="section">
```

Comment:
The background of the greeting sections should be write according to the Figma template.

Yours:
<img width="375" alt="image" src="https://user-images.githubusercontent.com/87897753/159585857-48c49cbc-7659-46d9-b1ff-4498f9bba59a.png">

Figma's
<img width="247" alt="image" src="https://user-images.githubusercontent.com/87897753/159585936-ce4985d4-c2b0-462c-a0f1-c044193bcaa7.png">